features/step_definitions/assertion_steps.rb

```diff
@@ -6,6 +6,18 @@
     expect(current_path).to eq new_user_registration_path
 end
 
-When('I fill in {string} field with {string}') do |field, input|
-    fill_in field, with: input
-end
\ No newline at end of file
+Then('I should be on the login page') do
+    current_path.should == new_user_session_path
```

Comment:
Should is deprecated please use `expect(page).to etc`

Reply:
Done